From 9d1403971f57b87d03aae6e02a57a6ab36e949d4 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Fri, 28 Sep 2018 13:58:17 +0800
Subject: [PATCH] arm64: rockchip_linux_defconfig: build the wifi as module by
 default

As the external/rkwifibt will callback the wifi module to instead of build-in
for linux.

Change-Id: Id6daabc21c06fb48e6fc1a74cc3d6ad1db6898f1
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/configs/rockchip_linux_defconfig | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/configs/rockchip_linux_defconfig b/arch/arm64/configs/rockchip_linux_defconfig
index 8095d4486aba..8d3b700e88ea 100644
--- a/arch/arm64/configs/rockchip_linux_defconfig
+++ b/arch/arm64/configs/rockchip_linux_defconfig
@@ -180,11 +180,12 @@ CONFIG_USB_NET_CDC_MBIM=y
 CONFIG_LIBERTAS_THINFIRM=y
 CONFIG_USB_NET_RNDIS_WLAN=y
 CONFIG_WL_ROCKCHIP=y
+CONFIG_WIFI_BUILD_MODULE=y
 CONFIG_WIFI_LOAD_DRIVER_WHEN_KERNEL_BOOTUP=y
-CONFIG_AP6XXX=y
-CONFIG_RTL8188EU=y
-CONFIG_MWIFIEX=y
-CONFIG_MWIFIEX_SDIO=y
+CONFIG_AP6XXX=m
+CONFIG_RTL8188EU=m
+CONFIG_MWIFIEX=m
+CONFIG_MWIFIEX_SDIO=m
 CONFIG_INPUT_FF_MEMLESS=y
 # CONFIG_INPUT_MOUSEDEV is not set
 CONFIG_INPUT_EVDEV=y
-- 
2.35.3

